aws/cft.yml

```diff
@@ -0,0 +1,141 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Conjur CE Stack, backed by a PostgreSQL database on RDS
+
+Parameters:
+  ConjurAccount:
+    Type: String
+    Description: Name of Conjur account
+
+  ConjurAdminPassword:
+    Type: String
+    NoEcho: true
+    Description: Password for Conjur admin account
+
+  DBAdminUser:
+    Type: String
+    Description: RDS instance admin user
+    Default: conjuradmin
+
+  DBAdminPassword:
+    Type: String
+    NoEcho: true
+    Description: RDS instance admin password
+
+  ImageId:
+    Type: AWS::EC2::Image::Id
+    Description: Conjur CE AMI
+
+  KeyName:
+    Type: AWS::EC2::KeyPair::KeyName
+    Description: AWS keypair to use
+
+  VpcId:
+    Type: AWS::EC2::VPC::Id 
+    Description: VPC in which to launch Conjur instances
+
+  VpcSubnetIds:
+    Type: "List<AWS::EC2::Subnet::Id>"
+    Description: Subnets in which to launch Conjur and Postgres. Specify two, in different Availability Zones.
+
+
+Metadata:
+  AWS::CloudFormation::Interface: 
+    ParameterGroups: 
+      - Label: 
+          default: "Conjur Configuration"
+        Parameters:
+          - ConjurAccount
+          - ConjurAdminPassword
+          - DBAdminUser
+          - DBAdminPassword
+      - Label: 
+          default: "EC2 Instance Configuration"
+        Parameters:
+          - ImageId
+          - KeyName
+          - VpcId
+          - VpcSubnetIds
+
+Resources:
+  ConjurInstance:
+    Type: "AWS::EC2::Instance"
+    Properties: 
+      ImageId: !Ref ImageId
+      InstanceType: t2.medium
+      KeyName: !Ref KeyName
+      SecurityGroupIds: [!Ref ConjurSecurityGroup]
+      SubnetId: !Select [0, !Ref VpcSubnetIds]
+      UserData:
```

Comment:
This works for now, but we'll want to clean it up. Thanks for getting this going!

Reply:
What about this needs cleaning up?